File: Examples/Features/Worksheets/ActivateWorksheet.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.Worksheets
{
    public class ActivateWorksheet : ExampleBase
    {
        public override void Execute(GrapeCity.Documents.Excel.Workbook workbook)
        {
            IWorksheet worksheet = workbook.Worksheets.Add();
            //Activate new created worksheet.
            worksheet.Activate();
        }
    }
}

File: Examples.VB/Features/Worksheets/ActivateWorksheet.vb
Namespace Features.Worksheets
    Public Class ActivateWorksheet
        Inherits ExampleBase
        Public Overrides Sub Execute(workbook As Excel.Workbook)
            Dim worksheet As IWorksheet = workbook.Worksheets.Add()
            'Activate new created worksheet.
            worksheet.Activate()
        End Sub
    End Class
End Namespace
